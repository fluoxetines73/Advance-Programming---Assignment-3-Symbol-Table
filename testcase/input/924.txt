INSERT x number
BEGIN
LOOKUP x
PRINT
ASSIGN x 1001
BEGIN
BEGIN
RPRINT
PRINT
RPRINT
INSERT d string
BEGIN
END
RPRINT
RPRINT
RPRINT
END
LOOKUP x
ASSIGN x 1001
INSERT d string
RPRINT
RPRINT
END
END
PRINT
RPRINT