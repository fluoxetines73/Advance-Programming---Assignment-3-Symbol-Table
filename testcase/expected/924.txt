success
0
x//0
success
x//0
x//0
x//0
success
d//3 x//0
d//3 x//0
d//3 x//0
0
success
success
d//2 x//0
d//2 x//0
x//0
x//0